docs/running-on-yarn.md

```diff
@@ -196,6 +196,14 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   It should be no larger than the global number of max attempts in the YARN configuration.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.submit.waitAppCompletion</code></td>
+  <td>true</td>
+  <td>
+  Whether to wait for application to complete in cluster mode. If set this to true, client process will enter a loop to track application's status.
```

Comment:
Perhaps reword slightly:
In YARN cluster mode, controls whether the client waits to exit until the application completes.  If set to true, the client process will stay alive reporting the application's status. Otherwise, the client process will exit after submission.